### ISSUE TITLE
About reference fields

### ISSUE THREAD

**Issue by ronaldw97** [web2py/py4web#78]
I was trying to make a CRUD with Publisher.It works but I have some problems with reference fields.Can´t edit reference fields.I clicked the id and it displays all fields, but I cant edit references.

**Comment by mdipierro** [comment]
Yes. This is a limitation of py4web dashboard at this time. working on it.
